NuCivic/dkan
# Add step to change timezone when testing

Issue: CIVIC-6268

## Description

Client sites may change the default timezones on their sites and certain timezones will cause one of the tests to fail on dataset.editor.feature

## How to reproduce

1.  Change the default timezone to America/Chicago
2. Run the `Replace node changed date with harvest source modified for harvested datasets` scenario from `test/features/dataset.editor.feature`
3. Confirm the test fails on `And I should see "2015-01-02"`

## QA Steps

- [ ] Run the steps above and confirm that the test now passes.

## Reminders
